data-explorer/error-codes.md

```diff
@@ -10,10 +10,10 @@ ms.date: 11/11/2020
 ---
 # Ingestion error codes in Azure Data Explorer
 
-The following list contains error codes you may come across during [ingestion](ingest-data-overview.md). When you enable failed ingestion [diagnostic logs](using-diagnostic-logs.md#ingestion-logs-schema) on your cluster, you can see error codes in the **Failed ingestion** operation log. You can alternatively monitor the **ingestion result** [metric](using-metrics.md#ingestion-metrics) to see the **Category** of ingestion errors, but not the specific error code. Errors below are organized by such categories. 
+The following list contains error codes you may come across during [ingestion](ingest-data-overview.md). When you enable failed ingestion [diagnostic logs](using-diagnostic-logs.md#ingestion-logs-schema) on your cluster, you can see error codes in the **Failed Ingestion Operation Log**. You can also monitor the **Ingestion Result** [metric](using-metrics.md#ingestion-metrics) to see the **Category** of ingestion errors, but not the specific error codes. Errors below are organized by these categories. 
```

Comment:
@Lea - according to the style guide, we capitalize using sentence case, not as you have done here. https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696
Does "Failed ingestion operation log" appear as such in the UI? Items in bold refer only to UI text.

Reply:
What do you mean by UI?
the term "Failed Ingestion Operation Log" appear as a term in the attached diagnostic logs link

Reply:
The UI is how terms appear in the user interface within the product.  You can see the formatting rules in the contributor's guide, here: https://review.docs.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=master#bold

As I see in this link: https://docs.microsoft.com/en-us/azure/data-explorer/using-diagnostic-logs?tabs=ingestion, failed ingestion either appears as "FailedIngestion" (camel case) or failed ingestion (lower case). It is not an official service name, so it is not appropriate to capitalize both words separately.  
![image](https://user-images.githubusercontent.com/62833511/101457635-72b2ad00-393e-11eb-8feb-64d3c07d8898.png)